webapp/src/test/resources/startup.properties

```diff
@@ -68,6 +68,7 @@ debug.retry.recorder.path=${user.dir}/target/retry
 debug.libext.feed.retention.paths=${falcon.libext}
 debug.libext.feed.replication.paths=${falcon.libext}
 debug.libext.process.paths=${falcon.libext}
+debug.extension.store.uri=jail://global:00/extension/store
```

Comment:
I see that config store is set to 
debug.config.store.uri=file://${user.dir}/target/store.

Instead of using jail FS can't we use file://${user.dir}/target/extension/store?